src/ImpactX.H

```diff
@@ -27,8 +28,14 @@ namespace impactx
         : public amrex::AmrCore
     {
       public:
-        ImpactX () = default;
+        ImpactX (amrex::Geometry const& geom, amrex::AmrInfo const& amr_info);
```

Comment:
I was wondering today, do we need to delete the default constructor then?
```suggestion
        ImpactX () = delete;
        ImpactX (amrex::Geometry const& geom, amrex::AmrInfo const& amr_info);
```